### PR TITLE
Update Torguard SHA

### PR DESCRIPTION
**Pull request by auanasgheps**
Manual update for Torguard SHA installer.
Shouldn't this be picked up by a bot?

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/11261)

### CODE REVIEW

**Comment by auanasgheps** [comment]
Sorry for the output not in english, but the manifest is validated and the install is successful.
What's wrong with this one?

image

**Review by auanasgheps — COMMENTED** [review]
Small change attempting to unlock the bot

**Comment by auanasgheps** [comment]
Not true, the bot is broken.

File: manifests/t/TorGuard/TorGuard/latest/TorGuard.TorGuard.yaml [hunk 1]
```diff
@@ -2,16 +2,16 @@ PackageIdentifier: TorGuard.TorGuard
 PackageVersion: latest
 PackageName: TorGuard
 Publisher: TorGuard
-License: Copyright Â© 2020 TorGuard.net
+License: Copyright © 2020 TorGuard.net
 Moniker: torguard
 Tags:
-- vpn openvpn
+- vpn openvpn wireguard
```
Comment (denelon) [inline manifests/t/TorGuard/TorGuard/latest/TorGuard.TorGuard.yaml:8]: These should be an array like:
 \- vpn
 \- openvpn
 \- wireguard

Comment (auanasgheps) [reply manifests/t/TorGuard/TorGuard/latest/TorGuard.TorGuard.yaml]: Ok, will fix. That's something new I didn't know.

**Review by auanasgheps — COMMENTED** [review]
Fix tags

**Comment by auanasgheps** [comment]
lol

**Comment by denelon** [comment]
It looks like we're getting errors from the server:

URI: https://torguard.net/, Validation result: Failed, Http status code: Forbidden 
URI: https://torguard.net/downloads/torguard-setup-latest.exe, Validation result: Failed, Http status code: Forbidden

**Comment by auanasgheps** [comment]
But if you open the URL from your machine, it probably can be downloaded successfully.

**Comment by jedieaston** [comment]
I already have a open issue for this at #10390. TorGuard blocks communication from Azure with a Access Denied or Forbidden HTTP code. (I attempted to download the installer in a GitHub Codespace and got the same codes as the pipeline got).

**Pull request closed without merging**